core/dom/range.js

```diff
@@ -2910,7 +2910,8 @@ CKEDITOR.dom.range = function( root ) {
 		 * @returns {CKEDITOR.dom.rect[]}
 		 */
 		getClientRects: ( function() {
-			if ( this.document.getSelection !== undefined ) {
+			'use strict';
```

Comment:
If we add `'use strict'` pragma, we should probably add it to the whole file, not this function only.

Reply:
I tried it, but... the `jshint` has some objectives 🤔 
![image](https://user-images.githubusercontent.com/6498567/162426749-e5f95b88-65c9-4781-adcb-9bd3ce40a1cd.png)

Reply:
Weird… 🤔  But I see that there is one pragma inside that file, inside the `execContentsAction()` function. Maybe `jshint` incorrectly interprets it as a pragma for the whole file?

Alternatively, we could just skip adding that pragma at all and just add to come up with some sensible way of testing the editor in the strict mode – yet, it seems quite cumbersome (https://github.com/ckeditor/ckeditor4/pull/5141#issuecomment-1094931387).

Reply:
> But I see that there is one pragma inside that file, inside the execContentsAction() function. Maybe jshint incorrectly interprets it as a pragma for the whole file?

Looks like you are right 😄 

So I removed the 'in functions' strict mode declarations & added one for the entire file.

The manual test seems to be still valid (fails without a fix, and passes with fix).

Reply:
But I needed to adjust it - since the error is visible in the console right away - even if we postpone editor creation.